### PR TITLE
Adding wizard to command storage del

### PR DESCRIPTION
**Pull request by MatheusClaudi**

### CODE REVIEW

**Review by lucasncabral — CHANGES_REQUESTED** [review]
Remember to add tests for the developed scenarios.

**Review by Vieira-Carlos — APPROVED** [review]

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -1122,6 +1122,7 @@ Options:
                                   sftph) [default: s3h]
   -g, --global                    Use this option to set configuration at
                                   global level
+  --wizard                        Enable the wizard to prompt the user requesting information when needed.
```
Comment (lucasncabral) [inline docs/mlgit_commands.md:1125]:
```suggestion
  -g, --global                    Use this option to set configuration at
                                  global level
  --wizard                        Enable the wizard to prompt the user 
                                  requesting information when needed.
```

File: tests/integration/test_03_add_store.py [hunk 2]
```diff
@@ -156,3 +157,14 @@ def test_12_add_sftph_storage_with_invalid_port(self):
...
+    @pytest.mark.usefixtures('switch_to_tmp_dir')
+    def test_13_del_storage_with_invalid_type(self):
+        invalid_type = 'not_a_type'
+        self.assertIn(output_messages['ERROR_STORAGE_TYPE_INPUT_INVALID'].format(invalid_type),
+                      check_output(MLGIT_STORAGE_DEL % BUCKET_NAME + ' --type=' + invalid_type))
+
+    @pytest.mark.usefixtures('switch_to_tmp_dir')
+    def test_14_del_storage_wizard_enabled_without_type(self):
+        self.assertIn(prompt_msg.STORAGE_TYPE_MESSAGE,
+                      check_output(MLGIT_STORAGE_DEL % BUCKET_NAME + ' --wizard'))
```
Comment (lucasncabral) [inline tests/integration/test_03_add_store.py:170]: We can increment this test, to validate that the command is correct. In addition to just checking if the message is being passed to the user, we can inform a value and check if the storage was really removed from the file.

File: docs/mlgit_commands.md (hunk 1 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: Suggestion was applied at [9268106](https://github.com/HPInc/ml-git/pull/235/commits/9268106103c2d858b401509f85540c570acc59d5).

File: tests/integration/test_03_add_store.py (hunk 2 above)
Comment (lucasncabral) [reply tests/integration/test_03_add_store.py]: Tests updated at [0b479a7](https://github.com/HPInc/ml-git/pull/235/commits/0b479a7dfcd973c440c732865e5593c34fb3aea2).

File: docs/mlgit_commands.md [hunk 3]
```diff
@@ -1122,6 +1122,8 @@ Options:
                                   sftph) [default: s3h]
   -g, --global                    Use this option to set configuration at
                                   global level
+  --wizard                        Enable the wizard to prompt the user
```
Comment (andersonpablosilva) [inline docs/mlgit_commands.md:1125]: Please, update the help message according to 9a4f421 - https://github.com/HPInc/ml-git/pull/237#discussion_r912151927

**Review by andersonpablosilva — APPROVED** [review]

File: docs/mlgit_commands.md (hunk 3 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: The message was updated at [9da1ccd](https://github.com/HPInc/ml-git/pull/235/commits/9da1ccd25e552f4a44658e4f00d360176b4b567e).

**Pull request merged**